src/rgw/rgw_main.cc

```diff
@@ -1241,7 +1241,11 @@ int main(int argc, const char **argv)
 
   /* setup openssl multi-threading functions */
   openssl_thread_setup();
-  
+  /* Initialise the library */
+  ERR_load_crypto_strings();
+  OpenSSL_add_all_algorithms();
+  OPENSSL_config(NULL);
+  srand ( time(NULL) );
```

Comment:
This is for random id we are appending for KMS Debugging. Only for staging. Will remove in prod